cmd/tracee-rules/internal/config/config.go

```diff
@@ -0,0 +1,89 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+type TraceeRulesCliConfig struct {
+	RegoConfig struct {
+		Aio               bool
+		EnableParsedEvent bool
+		PartialEval       bool
+		RunTimeTarget     string
+	}
+	Rules struct {
+		InputDirectory string
+		RuleIds        []string
+	}
+	Webhook struct {
+		Url         string
+		Template    string
+		ContentType string
+	}
+	Input struct {
+		File   string
+		Format string
+	}
+	Output struct {
+		Template string
+	}
+	Pprof struct {
+		Enable  bool
+		Address string
+	}
+	List bool
+}
+
+var Config TraceeRulesCliConfig
+
+func LoadConfig(fileName string) error {
+	cfgFile, err := os.Open(fileName)
+
+	if err != nil {
+		return err
+	}
+
+	defer cfgFile.Close()
+
+	byteValue, err := ioutil.ReadAll(cfgFile)
+
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(byteValue, &Config)
+
+	return err
+}
+
+func DefaultConfigLocation() string {
+	envLocation := os.Getenv("TRACEE_RULES_CONFIG")
+	if envLocation != "" {
+		return envLocation
+	}
+	return "./tracee-rules-config.json"
```

Comment:
Same as tracee-ebpf, this doesn't work as intended: the `.` refers to the working dir of the executing user instead of the location of the executable itself

Reply:
same as above